src/Microsoft.VisualStudio.ProjectSystem.Managed/StringComparers.cs

```diff
@@ -39,5 +39,10 @@ public static IEqualityComparer<string> DependencyProviderTypes
         {
             get { return StringComparer.OrdinalIgnoreCase; }
         }
+
+        public static IEqualityComparer<string> ItemTypes
```

Comment:
"Item types" are OrdinalIgnoreCase; `<compile Include="Foo.cs/>` and `<Compile Include="Foo.cs/>` are the same thing.

Reply:
Fixed.

Would one of the existing properties make sense for item types too? Does it warrant its own entry here?

Reply:
I'm fine with this, I like specific entries for this - requires less thinking when consuming it.